Tests/SourceKitLSPTests/SemanticTokensTests.swift

```diff
@@ -18,6 +18,7 @@ import SourceKitLSP
 import XCTest
 
 private typealias Token = SyntaxHighlightingToken
+private typealias Tokens = SyntaxHighlightingTokens
```

Comment:
I think we only have the typealias for `Token` because we use it *a lot* in this file. I would prefer to just spell out `SyntaxHighlightingTokens` instead. It makes the code easier to read because you don’t need to do a mental translation between `Tokens` and `SyntaxHighlightingTokens`.